tests/integration/test_option.py

```diff
@@ -0,0 +1,36 @@
+from guppylang.decorator import guppy
+from guppylang.module import GuppyModule
+from guppylang.std.option import Option, none, some
+
+
+def test_none(validate, run_int_fn):
+    module = GuppyModule("test_range")
+    module.load(Option, none)
```

Comment:
bit inconvenient to have to load these separately

Reply:
Yeah, but this is only an issue for explicit modules